### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by smartspot2** [csmberkeley/csm_web#428]
Currently, scheduler does not have any accessibility information built-in; see https://accessibility.18f.gov for guidance on accessibility.

Some major points:
* Contrast between colors is horrible ([reference](https://accessibility.18f.gov/color/))
* Buttons, links, and other interactions are not keyboard-focusable ([reference](https://accessibility.18f.gov/keyboard/))
* Page titles are not updated between subpages ([reference](https://accessibility.18f.gov/page-titles/))
* `aria-*` attributes are not specified

The site should be looked over, and various accessibility attributes should be added. This is also a good opportunity to add other kinds of descriptive text to all interactive elements. Colors should also be reworked to increase contrast.
